Frontend/android-native/mobility-customer/src/main/java/in/juspay/mobility/customer/MobilityCustomerBridge.java

```diff
@@ -579,7 +570,7 @@ public void run() {
                         JSONArray zonePoints = null;
                         zonePoints = new JSONArray(points);
                         for (int i = 0; i < zonePoints.length(); i++) {
-                            drawMarkers((Double) zonePoints.getJSONObject(i).get("lat"), (Double) zonePoints.getJSONObject(i).get("lng"), "");
+                            addZoneMarker((Double) zonePoints.getJSONObject(i).get("lat"), (Double) zonePoints.getJSONObject(i).get("lng"), (String) zonePoints.getJSONObject(i).get("place"), "ny_ic_zone_pickup_marker_yellow");
```

Comment:
`ny_ic_zone_pickup_marker_yellow` make sure that this image is available in customer module also.

Reply:
its in main

Reply:
we need it in customer module also.